src/lib/ruby-to-blocks-converter/ev3.js

```diff
@@ -89,6 +89,40 @@ const EV3Converter = {
                 }
                 break;
             }
+        } else if ((this._isSelf(receiver) || receiver === Opal.nil) &&
+        name === 'when' &&
+        args.length === 2 && args[0].type === 'sym' &&
+        rubyBlockArgs && rubyBlockArgs.length === 0 &&
+        rubyBlock) {
```

Comment:
【修正依頼】こんな感じでインデントを修正してほしいです。
```suggestion
        } else if ((this._isSelf(receiver) || receiver === Opal.nil) &&
                       name === 'when' &&
                       args.length === 2 && args[0].type === 'sym' &&
                       rubyBlockArgs && rubyBlockArgs.length === 0 &&
                       rubyBlock) {
```